go.mod

```
@@ -1,10 +1,10 @@
 module github.com/ramendr/ramenctl
 
-// Match kubectl-rook-ceph
-go 1.23.0
+// Required minimum version, must be available in downstream builders.
+go 1.23.5
```

```suggestion
go 1.23.5
```
I think it should be good as we have same minimum version